loadtests/loadtest/__init__.py

```diff
@@ -69,6 +71,16 @@ def __init__(self, *args, **kwargs):
         # Create at least some records for this user
         self.nb_initial_records = random.randint(3, 100)
 
+    def _get_configuration(self):
+        # Loads is removing the extra information contained in the ini files,
+        # so we need to parse it again.
+        config_file = self.config['config']
+        # When copying the configuration files, we lose the config/ prefix so,
+        # try to read from this folder in case the file doesn't exist.
+        if not os.path.isfile(config_file):
+            config_file = os.path.basename(config_file)
+        return Config(config_file).get_map('loads')
```

Comment:
here I would test again if the file exists before calling Config(), and raise an error if the file is not found. That'll make the problem clearer and != from a file existing but missing a [loads] section